src/accordionpane/README.md

```diff
@@ -0,0 +1,53 @@
+# @dojo/widgets/accordionpane/AccordionPane widget
+
+Dojo 2's `AccordionPane` component can be used to show multiple pieces of content inside collapsible panes within a common parent component. It provides responsive styling and a mechanism of control over multiple child `TitlePane` components.
+
+## Features
+
+- Optionally supports multiple panes open at once
+
+![Image of basic dialog](http://placekitten.com/450/300)
```

Comment:
Not a dialog, this is accordion pane and this should not be a place kitten

Reply:
I removed the picture entirely since our other components either have no picture or a placekitten picture. These READMEs are not thorough or entirely standardized yet.